src/lib.rs

```diff
@@ -1502,15 +1507,16 @@ impl Currency {
     }
 
     /// Create a currency instance from a ISO 4217 numeric code
-    /// 
+    ///
     /// # Example
-    /// 
+    ///
     /// ```
     /// use iso_currency::Currency;
-    /// 
+    ///
     /// assert_eq!(Currency::from_numeric(978), Some(Currency::EUR));
     /// ```
     pub fn from_numeric(numeric_code: u16) -> Option<Currency> {
+        #[allow(clippy::zero_prefixed_literal)]
```

Comment:
You already added it to the crate root, but IMO it's better to disable lints on a restricted scope?
Do you want me to remove this or the root one?

Reply:
Yeah, you're right let's remove the root level one.